thorlcr/thorutil/thmem.cpp

```diff
@@ -229,6 +243,8 @@ class CSpillableStreamBase : public CSimpleInterface, implements roxiemem::IBuff
     }
     virtual bool freeBufferedRows(bool critical)
     {
+        if (spillFile) // if spilt already. NB: thread-safe, as only set by this thread and can't this freeBufferedRows() can't be called on multiple threads concurrently.
```

Comment:
Not sure what the comment is trying to say but "and can't this freeBufferedRows() can't" does not make sense.